opencog/nlp/learn/README

```diff
@@ -141,27 +143,25 @@ So, set up the text-ingestion pipeline:
    (add-to-load-path "/usr/local/share/opencog/scm")
    (add-to-load-path ".")
 ```
+5) Copy all files from the `/opencog/nlp/learn/run` directory to your working
+   directory. Review and edit the language configuration inside 
+   `observe-launch.scm` if necessary (you can choose the appropriate 
+   'opencog-??.conf' file, if not using English).
+   This file starts the cogserver and sets a default prompt: set up by default
+   to avoid conflicts and confusion, and to allow multiple languages to be 
+   processed at the same time. You need to pass as arguments the database
+   credentials. See next step.
 
-6) Select a language, and copy one of the `pair-count-??.scm` files
-   from the `run` directory to your working directory. Review and edit
-   the configuration as necessary. It contains the REPL port number,
-   and a default prompt: These are set up by default to avoid conflicts
-   and confusion, and to allow multiple languages to be processed at
-   the same time.  It also contains the database credentials -- that's
-   probably the only thing you need to change.
-
```

Comment:
what happened to step 6? The configurations are somewhat different depending on the language you choose to work with.